app/controllers/callback_controller.rb

```diff
@@ -1,24 +1,12 @@
 class CallbackController < ApplicationController
   include ShopifyApp::WebhookVerification
 
-  def webhook
-    CallbackWebhookJob.perform_later(
-      shop_domain: shop.shopify_domain,
-      callback: params[:callback].to_unsafe_h,
-      topic: params[:topic]
-    )
-
-    head :ok
-  rescue => e
-    Bugsnag.notify(e)
-    head :ok
-  end
-
   def receive
     CallbackWebhookJob.perform_later(
       shop_domain: request.headers.fetch('X-Shopify-Shop-Domain'),
       callback: params.fetch(:callback).to_unsafe_h,
-      topic: request.headers.fetch('X-Shopify-Topic')
+      topic: request.headers.fetch('X-Shopify-Topic'),
+      shopify_identifier: request.headers.fetch('X-Shopify-Webhook-Id')
```

Comment:
I am worried to see the code here grow and grow over time.

Next thing to add will be `X-Hooklys-Id` header and so on.

We could add all headers `headers: headers`? 

One one hand it will mean no longer have to change the callback controller code, on the other it is a waste of memory (which I do not think we care at this point).

Thoughts? I'm good if we want to keep this as-is for now.

```suggestion
      headers: request.headers.to_unsafe_h
```

Reply:
If we need hooklys id, let's add hooklys id? I'd still prefer to be explicit as opposed to send all of them, although 90% are waste?